src/start.sh

```diff
@@ -101,16 +101,25 @@ sed -i "s/59 17/$((1 + RANDOM % 58)) $((12 + RANDOM % 8))/" /crontab.txt
 
 /usr/sbin/crond
 
+#migrate Database if needed:
 gravityDBfile=$(getFTLConfigValue files.gravity)
-
-if [ -z "$SKIPGRAVITYONBOOT" ] || [ ! -f "${gravityDBfile}" ]; then
-    if [ -n "$SKIPGRAVITYONBOOT" ];then
-        echo "  SKIPGRAVITYONBOOT is set, however ${gravityDBfile} does not exist (Likely due to a fresh volume). This is a required file for Pi-hole to operate."
-        echo "  Ignoring SKIPGRAVITYONBOOT on this occaision."
-    fi
+if [ -n "${SKIPGRAVITYONBOOT}" ]; then
+  if [ -f "${gravityDBfile}" ]; then
+    #skip set + file =>update if needed
+    echo "  Skipping Gravity Database Update."
+    source /opt/pihole/utils.sh
```

Comment:
What is `utils.sh` being sourced for here?

Reply:
> What is `utils.sh` being sourced for here?

At a time, I found it was mandatory, but now I commented it and it's not anymore needed.... I guess it can be removed.

on a side note, return is not allowed when not in a function: (uid/gid changes).
`[[ ${currentId} -eq ${PIHOLE_UID} ]] && return`

Reply:
Thanks, good catch, I'll address that in a follow up PR (I should have waited for a review really, but.. sometimes I am impatient)